src/liballoc/tests/binary_heap.rs

```diff
@@ -347,7 +347,7 @@ fn assert_covariance() {
 // Destructors must be called exactly once per element.
 // FIXME: re-enable emscripten once it can unwind again
 #[test]
-#[cfg(not(any(miri, target_os = "emscripten")))] // Miri does not support catching panics
+#[cfg(not(target_os = "emscripten"))]
```

Comment:
I think emscripten now supports catching panics (see https://github.com/rust-lang/rust/pull/65832)

Reply:
That should be a separate PR though from a CI perspective.

Reply:
Cc @tlively looks like some tests have not been fully enabled for unwinding being fixed in emcscripten. I recommend auditing `rg emscripten src/lib{core,alloc}`.